src/libs/elektra/kdb.c

```diff
@@ -1130,6 +1142,7 @@ int kdbGet (KDB * handle, KeySet * ks, Key * parentKey)
 	if (cacheData && handle->globalPlugins[POSTGETCACHE][MAXONCE])
 	{
 		splitCacheStoreState (handle, split, handle->global, cacheParent, initialParent);
+		elektraCutProc (ks); // remove proc keys before caching
```

Comment:
Again, haven't tested, but doesn't this line mean that the `proc` keys won't be returned by `kdbGet`. As far as I can tell from the code they are removed, but not added again.

Reply:
You're right. I thought the proc keys are only internal. I'll fix it.